pom.xml

```diff
@@ -59,8 +59,50 @@
     <maven.gpg.plugin.version>1.6</maven.gpg.plugin.version>
   </properties>
 
```

Comment:
I think we're missing a change where the parent section in the pom is removed..?

Reply:
Ah yes :)